cli/templateinit.go

```diff
@@ -24,8 +24,9 @@ func templateInit() *cobra.Command {
 			exampleNames := []string{}
 			exampleByName := map[string]examples.Example{}
 			for _, example := range exampleList {
-				exampleNames = append(exampleNames, example.Name)
-				exampleByName[example.Name] = example
+				name := fmt.Sprintf("%s\n      %s", example.Name, example.Description)
```

Comment:
Just curious, why not `\t`?

Reply:
A very valid question, my reasoning was to have predictable alignment. The select statement uses 4 spaces for alignment but counting from the cursor (`>`) it seems like 2 spaces, hence `4 + 2`.

Reply:
Aha, makes sense! Thank you.